static/app/components/feedback/feedbackItem/feedbackItem.tsx

```diff
@@ -57,6 +60,17 @@ export default function FeedbackItem({feedbackItem}: Props) {
             </Flex>
             <Flex align="center" gap={space(0.5)}>
               <ProjectAvatar project={project} size={12} /> {slug}
+              {feedbackItem.replay_id ? (
+                <Link
+                  to={{
+                    pathname: normalizeUrl(
+                      `/organizations/${organization.slug}/replays/${feedbackItem.replay_id}/`
+                    ),
+                  }}
+                >
+                  {getShortEventId(feedbackItem.replay_id)}
+                </Link>
+              ) : null}
```

Comment:
I'm not sure we should be putting the replay id up here... should this be the feedback_id itself?

Reply:
i couldn't tell based on jesse's design but `feedback_id` makes more sense